stl/inc/optional

```diff
@@ -221,19 +224,23 @@ public:
     constexpr optional(nullopt_t) noexcept {}
 
     template <class... _Types, enable_if_t<is_constructible_v<_Ty, _Types...>, int> = 0>
-    constexpr explicit optional(in_place_t, _Types&&... _Args) : _Mybase(in_place, _STD forward<_Types>(_Args)...) {}
+    constexpr explicit optional(in_place_t, _Types&&... _Args) noexcept(
+        is_nothrow_constructible_v<_Mybase, in_place_t, _Types...>)

```

Comment:
This is saying `in_place_t` instead of `const in_place_t&` which would exactly match the argument. However, this doesn't make a difference for `_Mybase`.

What would be simpler is writing this as `is_nothrow_constructible_v<_Ty, _Types...>`, bypassing the `_Mybase` layer. Occurs below.